src/components/Docs/Docs.tsx

```diff
@@ -0,0 +1,17 @@
+import { Text } from '@chakra-ui/react';
+
+export function DocsURL({ url, children }: { url: string | undefined, children: React.ReactNode; }) {
+  if (!url) {
```

Comment:
`url` looks like it should be required. This won't make sense if nothing is returned and the children are also not rendered.